lib/web_ui/dev/steps/run_tests_step.dart

```diff
@@ -164,6 +173,47 @@ class RunTestsStep implements PipelineStep {
     }
     return message.toString();
   }
+
+  Future<SkiaGoldClient?> _createSkiaClient() async {
+    final SkiaGoldClient skiaClient = SkiaGoldClient(
+      environment.webUiSkiaGoldDirectory,
+      browserName: browserName,
+    );
+
+    if (!await _checkSkiaClient(skiaClient)) {
+      print('WARNING: Unable to use Skia Client in this environment.');
+      return null;
+    }
+
+    return skiaClient;
+  }
+
+  /// Checks whether the Skia Client is usable in this environment.
+  Future<bool> _checkSkiaClient(SkiaGoldClient skiaClient) async {
+    // Now let's check whether Skia Gold is reachable or not.
+    if (isLuci) {
+      if (SkiaGoldClient.isAvailable) {
```

Comment:
Nit: This could be `&&` for easier reading

Reply:
If we do `&&` it will mess up the `else` branch below.